### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MyroslavSynenko**
-DEMO LINK https://myroslavsynenko.github.io/react_static-list-of-posts-js/

### CODE REVIEW

File: src/components/CommentInfo/CommentInfo.jsx [hunk 1]
```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
+
```
Comment (OleziO) [inline src/components/CommentInfo/CommentInfo.jsx:7]:
```suggestion
      <span> {' by '} </span>
```

File: src/components/CommentList/CommentList.jsx [hunk 2]
```diff
@@ -1,3 +1,17 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0 ? (
```
Comment (OleziO) [inline src/components/CommentList/CommentList.jsx:6]:
```suggestion
    {comments.length ? (
```

File: src/components/PostInfo/PostInfo.jsx [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+
```
Comment (OleziO) [inline src/components/PostInfo/PostInfo.jsx:14]:
```suggestion
```

File: src/components/PostInfo/PostInfo.scss [hunk 4]
```diff
@@ -1 +1,16 @@
-//
+.PostInfo {
+  margin: 10px auto;
+  width: 90%;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: lightgray;
```
Comment (OleziO) [inline src/components/PostInfo/PostInfo.scss:6]: Use HEX colors instead of color names, cause they are different in different browsers

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**